code/src/components/TodoCounter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+/* import completed from 'reducers/completed'; */
+import uncompleted from 'reducers/uncompleted';
+
+export const TodoCounter = () => {
+  const dispatch = useDispatch();
+  const allUncompletedTasks = useSelector((store) => store.uncompleted.tasks);
+  const numberOfUncompletedTasks = allUncompletedTasks.length;
```

Comment:
Instead of `numberOfUncompletedTasks`, you can use a more descriptive name like `uncompletedTaskCount` to make it clearer what the variable represents.